Snowflake.Data/Core/Session/SFSessionProperty.cs

```diff
@@ -110,7 +110,9 @@ internal enum SFSessionProperty
         [SFSessionPropertyAttr(required = false, defaultValue = "60m")]
         EXPIRATIONTIMEOUT,
         [SFSessionPropertyAttr(required = false, defaultValue = "true")]
-        POOLINGENABLED
+        POOLINGENABLED,
+        [SFSessionPropertyAttr(required = false, defaultValue = "false")]
+        DISABLE_SAML_URL_CHECK

```

Comment:
Please add description to https://github.com/snowflakedb/snowflake-connector-net/blob/master/doc/Connecting.md

Reply:
Added description